examples/project_ENGIE.py

```diff
@@ -124,7 +124,7 @@ def prepare(self):
                     "time"                 : "time",
                     "Wind_turbine_name"    : "id",
                     "Power_W"              : "wtur_W_avg",
-                    "Ws_avg"               : "wmet_HorWdSpd_avg", 
+                    "Ws_avg"               : "wmet_wdspd_avg", 
```

Comment:
Shouldn't we switch all of these to lower case, then?

Reply:
Yeah I was thinking the same, I just didn't want to inadvertently break some other codes :)

Reply:
We use all lower case. Here is the schema file:

https://github.com/NREL/OpenOA/blob/develop/operational_analysis/types/plant_schema_25.json

Reply:
I have updated most of them as lower cases. For now, I have kept wtur_W_avg and wmet_wDir_avg, that are used in this way in some examples and other classes/scripts...

Reply:
That's fine with me to stick with the precedent for these variable names. I do think the readability suffers when they're all lowercase though.